src/components/issues/IssuesPage.js

```diff
@@ -68,11 +73,29 @@ const Wrapper = styled.div`
   margin: 70px auto 0;
 `;
 
+const Loader = styled.div`
+  display: none;
```

Comment:
you can just use conditional rendering and avoid the whole display: block/none logic inside the styled component